pkg/engine/engine.go

```diff
@@ -282,6 +283,17 @@ func (e *Engine) DoRun(ctx context.Context, testplan string, testcase string, ru
 		return nil, fmt.Errorf("error while generating test run ID: %w", err)
 	}
 
+	// Add the default parameters set on the manifest.
+	for paramName, paramValue := range tcase.Parameters {
+		if _, ok := input.Parameters[paramName]; !ok {
+			data, err := json.Marshal(paramValue.Default)
```

Comment:
(Δ)

Reply:
I am marshalling this as a JSON value so it can be used on the environment variables passed to docker so we can get them afterwards using `GetCurrentRunEnv`. This seemed to be the easier way to support this kind of things. If you have a different suggestion, I'll be glad to hear it :D